launch/tier4_perception_launch/launch/object_recognition/tracking/tracking.launch.xml

```diff
@@ -2,12 +2,28 @@
 <launch>
   <arg name="publish_rate" default="10.0"/>
   <arg name="enable_delay_compensation" default="true"/>
+  <arg name="radar_tracker_input" default="/perception/object_recognition/detection/radar/far_objects"/>
+  <arg name="radar_tracker_output" default="/perception/object_recognition/tracking/radar/far_objects"/>
+  <arg name="use_radar_object_tracker" default="true"/>
```

Comment:
I want to change launch node by perception_mode, like detection
https://github.com/autowarefoundation/autoware.universe/blob/4f0a137f4fcaea00cd78024de8b7717956a60172/launch/tier4_perception_launch/launch/object_recognition/detection/detection.launch.xml#L40-L41

Reply:
I added `mode` arg to tracking.launch.
This enables single args to control both detection/tracking launch simultaneously.

Radar far object tracker is used only when `radar` input is enabled.

https://github.com/autowarefoundation/autoware.universe/pull/4361/commits/25e1b871880cb274bd1345bbf4dcafeb4a52dd03